app/models/manageiq/providers/amazon/storage_manager/ebs/refresh_parser_inventory_object.rb

```diff
@@ -59,6 +59,11 @@ def parse_snapshot(snap)
     }
   end
 
+  # Overridden helper methods, we should put them in helper once we get rid of old refresh
+  def get_from_tags(resource, item)
+    (resource['tags'] || []).detect { |tag, _| tag['key'].downcase == item.to_s.downcase }.try(:[], 'value')
+  end
+
```

Comment:
Once this PR is merged, the ```get_from_tags``` method will be overridden in 3 different RefreshParserInventoryObject subclasses (NetworkManager, CloudManager, StorageManager) - each one identical.
For the record, as the comment on line 62 indicates, the above implementation will replace the current version in the RefreshHelperMethods module when the RefreshParser class becomes obsolete.